### ISSUE TITLE
Bug: Pixel 6 Pro - Saturated Colors Display Setting Dims Screen

### ISSUE THREAD

**Issue by g1t3m** [GrapheneOS/os-issue-tracker#2249]
Platform: Google Pixel 6 Pro
OS Version: 13 (TQ2A.230505.002)
Issue: Selecting Saturated Colors under Display settings dims the screen significantly.

To Replicate:
1. Open Settings App
2. Select Display settings
3. Select Color settings
4. Select Saturated colors

Expectation: Display colors will change to Saturated and screen will stay at preset brightness.

Actual Result: Screen brightness dims significantly. Selecting any other color setting undoes the dimmed brightness bug.

Please let me know if you require additional info to resolve this issue.

**Comment by TheLoneDev** [comment]
Do you have screen dimming on?
It reminds me another bug that Saturated causes, which dimming can only be enabled, but not disabled.

**Comment by g1t3m** [comment]
Yes, in that case I did have screen dimming on. I just tried it with screen dimming disabled and the brightness stayed the same.

**Comment by TheLoneDev** [comment]
Try rebooting after dimming is disabled.

**Comment by catdevnull** [comment]
probably related to #1437

**Comment by thestinger** [comment]
Is this still an issue with the latest release?

**Comment by TheLoneDev** [comment]
Also for me in Pixel 7 Pro

**Comment by thestinger** [comment]
@TheLoneDev On which release?

**Comment by TheLoneDev** [comment]
> @TheLoneDev On which release?

UQ1A.231205.015.2023120800
